client/post-editor/media-modal/index.jsx

```diff
@@ -427,7 +443,7 @@ export class EditorMediaModal extends Component {
 		if ( this.state.source !== '' ) {
 			buttons.push( {
 				action: 'confirm',
-				label: this.props.translate( 'Copy to media library' ),
+				label: selectedItems.length > 1 ? this.props.translate( 'Copy to media library' ) : this.props.translate( 'Insert' ),
```

Comment:
> AFAIK, the site icon modal gets its button name from here, and the set featured image from here - so this change doesn't affect them.

Yep, but what I mean is that you need to allow that prop to override the value here which is hardcoded to either 'Copy to media library' or 'Insert'. This is used elsewhere in the same function:

```js
	label: this.props.labels.confirm || this.props.translate( 'Insert' )
```

The text for the featured image and site icon is incorrect for me when viewing Google Photos. It is correct when viewing internal